packages/wallets/glow/src/adapter.ts

```diff
@@ -0,0 +1,266 @@
6ODnl5wwYZizAknR5U5hAALU8zJUNgVq0ISOsKzJPsYEamTaqXV157VeMgLmonz60zRvr4wH6pmoxNDSEKiedcVE2eG3g05nmB4TZ1mc/8mC99XZ1SWTFWXvzpT+U03rzkzqk30eXAwYNaK8dbGrz6NNxHC9WJ+6IpoVok3qOBw9cY2zRaDZBDAd8fd+yUqqqq3PmH6rt37dJ6HLs6ue1jz/HzEpLA5qjnbkiBGsV3UnydOpDslxROo4M4Z4zCpngEuzsiZrxOMm7eTmxV7e233tImMoVeyqMxAIGH7CJbyWAfMT+1EVgSUVIHqdapPnAA5fpnnpO/+tIVJgQhJ/AjZ3k0fnWDebWDt39SdkGFdBw/Ia1Hjc1wpnOlCWWlUlZVrWD5/M1f/1qWLFliJIl7G8edOHFC7v7mN2Xi2DLpOnUCKk+GQwoRfbRoAwZtS2YtDrsRU6Wg33yDQsKZ6JBCvDwbPaFMtu56V7Zs3iKLEY1qH+o8gOzCCXEmQvimmXNk2wc7ddwIXJf9wz9KXf0U3ewKsU9QJemdjsOwaRMvQJ1smj6lTnbv3SeHDx+WCQhHbHzG+Z/GC4nrb7hBZjfUy6mjbeqEuJfQ0qhJDOPzEE2UjC6FamGAC9WiehEFgRCESiXhIjTol4kyWh68d7XMe22Dvgayi73w3Hqlp/2Dj2TtAw8C6BKZXD1ZRuO0aHdu7ZBzoWE/gjM/1WkTJLEaITzTzzdulJW33qpAOLYNZxeCqKmsUGl4kIbaA2FYtQKdauik+7K5S8PuM10qEeqxAokWVkAw2uKxI+WtXdtk3aM/lJVfv1Wf/v7t38lfLF4k6/GSbelll0o5gsfcRCnkOgc+o95b3bd9SfAvXn1VQeDkqLEXXfod37gdLyIek+k1VdJ9qh1RA14NYbzG6VpGEsHroNGlZdxHjJg8uE3aBwVnwRCI44WSPN4jl86+RFbdfptMR0g/Z95c2fzmJuwfbTIeZ3cmS7gl9FxE5/ZjnX2oTl9btUqWNDfLK3jrWF19p6z/yXMK4qIZDXLy8J8Qb+UpTbyQNviPz2bazOXzvhB2d3ZLGjaCr1KYnOLiIlGODMvxABmuYW/rDlmz7nG5aeUtpEVPbfruiQP+H4lqykx1otp9777vyP0PPiSzaquk98xx2CJUHskw2EiUDKBz8iGlPEqkpJQ2e1Yi5swAe8FrSmY/ypSWG2AgyqmT58oPVt0jb/7qDV2ABDA0Ga5G+vB/udgxnIPq9DwkQRBzaifJYE+H2ZPAH08J54ZqJAGe/rlkQHbgq7GDUAfEM4Ng3aFREgRB6be7VKieohx6+/m//KI8uvb7OEV2wJtALbEYAZG7Vs2G42C7lQDrHENGUkXv/Ofb5as33CgX1VVJqrcTIXoGNJDzjBAMCNZhC9jxIlBD7RJ4UyfUrwzS2SJ+8UQ7XvGa0EJ3eSgZRWjaMAkW5emSkptRXSf/+fQT8vuNm6Rs4ngYewVcIXQZ420mEMt125ZbnsQ+8TJc8SJ4u23/9Z4smj1d+k4fw3hwHmuR87QHE9oQNDLuefRQ1XKdAK9WHRzo2n2oT2siFq9ID6RoIKAZQKiRnASEYAzusPGwbrwBavjidLpXmi5qlr6TZ+Tyv/6SfHnuxfK3t1wvc+bPl/EXTpBifPThoYjz2ZRCANjZ2SmHDh6QrXiz+Phdd8tePFxQXSk+IoOeY61q2CFDc02kmhVaiEnqtdiCBihCkIgDgeceQNjjvSPxxCVJLwmfFe3a7ISeBr3hig0F7MSUd7ZnQApw6Lm26XPS++lJWX3LCmEoOLuqTmoXLZSxEyolHx96GAF3d3XLkYOtsv3Fl8RsnfiHhKpxMmFUqfR1nJR0Tx9OmPnaV6mMCLeFBUIArLOE6oWJBL52+c425/rLljf19w6809PRHfDTLxAb9UJH3UU5EJLgYBvbELDJPIjhux4iAjcOv14QR6yYlQGcH7r37pZu9CIwaCPegIhw1y+qmyRxvC7ix6JkdwdC8wHD2mjfIWiTbBndojAgKB1qjMvPb0FJ2RgXnuti0ifXXXLtlnQytbivpzuNk1dMdTIColLRSQjGxjYGSMivR0omDZwnNYDCDy6OIsUehBHQSbbxk14Gn+oymRQM3nyjZLueV3iNgLDlbIrqZCZp0AcEoU4iPWrk6FheUfGWn2x+r5mMwreIxJ1wsH/IDqY8vMzG2rlSiTgxJBVzFlCJkEWaCBEJFwOXgSqIRawWhngNC8B6+AHYGIYEUEt6Fh5C4cfQh2BRj8azFdU/S3Y1MDrAiw4vH84lLxG7ix29FfNWxP5t64/a5lXPyRbnF16KUDsNzC68Ar54wTsgqwdRL2I9h3GLVD3rSdTDgDjjaYzLxBDz3JbsD2psNn1NH7pYM595zntqhmoHSs6l954bxmLxTFlpCaw8du9/vPnu+pbmZnplkRbBP6hIS7Dic19dh//+WIkzMlQ9kwWffSymjocdja2A5+QweEg5OFAtw3GWlACyqgzrkAp/qnLmWRZSMW1GHdmX9qIqBpFENaxmZYInRhR4ueNmAMLDS2s36/iPPf2bnatagBsZL7DMECqJjly19KZ73CBcwxPdQDKJNfAPIZgKz8EXvDCiiikQumQqBDI3F+aIYNoNiVW1Yp2EKihbj8CwD3tyI1Uo5mpAKWX4P4MwgKRCBI0e/hHB9fE9xoEk/v2NnWvYA4kYIqysoeG70uJQMv/yxa814iPIWpyzlvAogw/wMFYcWWGkKgm1F3Pwt0BUKpQOMwARjNYViGmDyYPgs2AUBAGQATR2+2Mda1CleZjiPxSEiDBCx3vL9+N3Pv76e++2gLH3GQAqAJUIVh1KtJkntj+RZsO/Xn7rQhC+zAlClGE1pFFuGBQZvEqGRFMqRjL4fygFYIBQEuaeAAzBRhpnVcpIRb0WRlJ6TJQEbk+ErtMKY30HZvvSY7/csY3PVuAfgZ7APwKxbtN/AyEn1KgMAwQMAAAAAElFTkSuQmCC';
```

Comment:
Thought: This isn't about you – you're just replicating what came before – but we might consider out-lining these inlined icons at some point.

Pro: inlining them is simple and they load instantly.
Con: every visitor pays the cost of the images, on every visit, whether or not they ever open the wallet selection dialog. This icon here, for instance, is 5.7K gzipped. Multiply that by the number of wallets a given app supports and you have a whole hamburger.

Reply:
One thing you might consider here – if possible – is to drum up an SVG version of this icon. Since it's so geometric, it might very well be tiny.

Reply:
Yea let me ping our designers and see if they can get me an svg version.